src/command.rs

```diff
@@ -13,13 +13,33 @@ pub fn authenticate(spotify_client_id: &str, spotify_client_secret: &str) {
     }
 }
 
-/// Create a Spotify playlist
+/// Create a Spotify playlist.
 pub fn create_playlist(spotify_client_id: &str, spotify_client_secret: &str, matches: &ArgMatches) {
     let name = matches.value_of("name").unwrap();
     let spotify_wrapper =
         spotify::SpotifyWrapper::new(&spotify_client_id, &spotify_client_secret).unwrap();
-    let playlist = spotify_wrapper.create_playlist(&name).unwrap();
-    dbg!(&playlist);
+    let playlist = match spotify_wrapper.create_playlist(&name) {
+        Ok(p) => p,
+        Err(e) => {
+            eprintln!("{}", e);
```

Comment:
For improved print debugging :-) you can use the debug format specifier i.e. `eprintln!("{:?}", e);`

Reply:
OK wait you actually want the output to go to the user. Nevermind

Reply:
Yeah, but I do want to come back to the error handling so I can improve output to the user.